app/controllers/admin/questions_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Admin::QuestionsController < ApplicationController
```

Comment:
Let's make sure to add authorization/authentication requirements to accessing this page.

Reply:
I set the Questions controller to inherit from AdminController, which I believe adds what you mentioned here.